playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -586,10 +586,22 @@ EDXAPP_CODE_JAIL_LIMITS:
 
 # Set the number of workers explicitly for lms and cms
 # Should be set to
-# EDXAPP_WORKERS:
-#   lms: <num workers>
-#   cms: <num workers>
+EDXAPP_WORKERS:
+  lms: 14
+  cms: 14
+# EDXAPP_WORKER_THREADS
+#   lms: <num worker threads>
+#   cms: <num worker threads>
 EDXAPP_WORKERS: !!null
+EDXAPP_WORKER_THREADS: !!null
+EDXAPP_WORKER_CLASS: !!null
+# Gunicorn will restart each worker after x amount of
+# requests processed. The x value will be randomly selected between
+# 1 and this value (default 1500)
+EDXAPP_CMS_MAX_REQUEST_JITTER: 1500
+EDXAPP_LMS_MAX_REQUEST_JITTER: 1500
```

Comment:
Will this override `EDXAPP_LMS_MAX_REQ`?

Is there some reason we don't want a specific number of requests, but instead a random number?

Reply:
@kaizoku Yes, using "jitter" is better because with a fixed number of request there's a chance all the workers will restart at the same time creating a bottleneck.
I still left the `(LMS|CMS)_MAX_REQUEST` because jitter is implemented only on 19.2+

Reply:
@kaizoku This will not override `EDXAPP_LMS_MAX_REQ`. By default `EDXAPP_LMS_MAX_REQ` is going to be used, unless it is not defined then `EDXAPP_LMS_MAX_REQUEST_JITTER` will be used.
This way nothing will change immediately when we merge this PR. Once we want to start using jitter across the board then we can gradually do the update on a cilent by client basis.

Reply:
Gotcha, thanks for the clarification.

Reply:
For reference, jitter in the context of gunicorn was misinterpreted here. Jitter is *added* to the max_request setting, with the following formula:

```
if max_requests:
  worker_max_requests = max_requests + randint(0, max_requests_jitter)
```

The idea of setting either max_requests xor max_requests_jitter was invalid and would have resulted in major issues like workers never restarting...